src/generated/get-webhook-payload-type-from-event.ts

```diff
@@ -1,215 +1,412 @@
-  "team.created": WebhookEvent<EventPayloads.WebhookPayloadTeam>;
-  "team.deleted": WebhookEvent<EventPayloads.WebhookPayloadTeam>;
-  "team.edited": WebhookEvent<EventPayloads.WebhookPayloadTeam>;
-  "team.removed_from_repository": WebhookEvent<EventPayloads.WebhookPayloadTeam>;
-  team_add: WebhookEvent<EventPayloads.WebhookPayloadTeamAdd>;
-  watch: WebhookEvent<EventPayloads.WebhookPayloadWatch>;
-  "watch.started": WebhookEvent<EventPayloads.WebhookPayloadWatch>;
-  workflow_dispatch: WebhookEvent<EventPayloads.WebhookPayloadWorkflowDispatch>;
-  workflow_run: WebhookEvent<EventPayloads.WebhookPayloadWorkflowRun>;
-  "workflow_run.action": WebhookEvent<EventPayloads.WebhookPayloadWorkflowRun>;
-  "workflow_run.completed": WebhookEvent<EventPayloads.WebhookPayloadWorkflowRun>;
-  "workflow_run.requested": WebhookEvent<EventPayloads.WebhookPayloadWorkflowRun>;
+export interface EmitterEventWebhookPayloadMap {
+  check_run: CheckRunEvent;
+  "check_run.completed": CheckRunCompletedEvent;
+  "check_run.created": CheckRunCreatedEvent;
+  "check_run.requested_action": CheckRunRequestedActionEvent;
+  "check_run.rerequested": CheckRunRerequestedEvent;
+  check_suite: CheckSuiteEvent;
+  "check_suite.completed": CheckSuiteCompletedEvent;
+  "check_suite.requested": CheckSuiteRequestedEvent;
+  "check_suite.rerequested": CheckSuiteRerequestedEvent;
+  code_scanning_alert: CodeScanningAlertEvent;
+  "code_scanning_alert.appeared_in_branch": CodeScanningAlertAppearedInBranchEvent;
+  "code_scanning_alert.closed_by_user": CodeScanningAlertClosedByUserEvent;
+  "code_scanning_alert.created": CodeScanningAlertCreatedEvent;
+  "code_scanning_alert.fixed": CodeScanningAlertFixedEvent;
+  "code_scanning_alert.reopened": CodeScanningAlertReopenedEvent;
+  "code_scanning_alert.reopened_by_user": CodeScanningAlertReopenedByUserEvent;
+  commit_comment: CommitCommentEvent;
+  "commit_comment.created": CommitCommentCreatedEvent;
+  content_reference: ContentReferenceEvent;
+  "content_reference.created": ContentReferenceCreatedEvent;
+  create: CreateEvent;
+  delete: DeleteEvent;
+  deploy_key: DeployKeyEvent;
+  "deploy_key.created": DeployKeyCreatedEvent;
+  "deploy_key.deleted": DeployKeyDeletedEvent;
+  deployment: DeploymentEvent;
+  "deployment.created": DeploymentCreatedEvent;
+  deployment_status: DeploymentStatusEvent;
+  "deployment_status.created": DeploymentStatusCreatedEvent;
+  fork: ForkEvent;
+  github_app_authorization: GithubAppAuthorizationEvent;
+  "github_app_authorization.revoked": GithubAppAuthorizationRevokedEvent;
+  gollum: GollumEvent;
+  installation: InstallationEvent;
+  "installation.created": InstallationCreatedEvent;
+  "installation.deleted": InstallationDeletedEvent;
+  "installation.new_permissions_accepted": InstallationNewPermissionsAcceptedEvent;
+  "installation.suspended": InstallationSuspendedEvent;
+  "installation.unsuspended": InstallationUnsuspendedEvent;
```

Comment:
These values are incorrect, I fixed these in octokit/webhooks#300
https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#installation

Reply:
Fixed in https://github.com/octokit/webhooks/pull/324